### PR TITLE
Lra 447 art survey display documents and images on the art item show page

### PR DESCRIPTION
**Pull request by britaumich**
I took the procedure to add and display documents from our VOD application.
(I added indigo colors to tailwind because they are used in VOD)

Images are displayed with the lightbox from this article:
[Building a Lightbox with Hotwire and Swiper.js](https://blog.minthesize.com/lightbox-hotwire-swiper)

### CODE REVIEW

**Review by rsmoke — CHANGES_REQUESTED** [review]
Uncomment the variant manager for the thumbnails in application.rb and set the thumbnail size to a fixed width to prevent stretching in the _show_images.html.rb. 
LRA-450 will improve on this too

File: config/application.rb [hunk 1]
```diff
@@ -33,5 +33,8 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    # run 'brew install vips' or uncomment the next line
+    # config.active_storage.variant_processor = :mini_magick
```
[the remark this answers is not in the document]
Comment (britaumich) [reply config/application.rb:38]: Rails 7 (or ruby 3, I don't understand that clearly) requires vips to process images. You can install vips ('brew install vips', for example) or uncomment line 28 to keep using mini_magick.

[the remark this answers is not in the document]
Comment (rsmoke) [reply config/application.rb:38]: because we don't have brew on Digital Ocean, probably uncommenting is the best solution. Uncommenting worked on my local

**Comment by britaumich** [comment]
The :thumb variant is defined in the model. They added some new features to process images in Rails 7 (see the article
https://www.bigbinary.com/blog/rails-7-adds-ability-to-use-predefined-variants). 
I added the lightbox following this article [Building a Lightbox with Hotwire and Swiper.js](https://blog.minthesize.com/lightbox-hotwire-swiper).

I tried to play with :thumb sizes, but unsuccessfully.

File: config/application.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (britaumich) [reply config/application.rb]: done

**Review by rsmoke — APPROVED** [review]

**Pull request merged**